shell/platform/linux/fl_settings_plugin.h

```diff
@@ -0,0 +1,44 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_LINUX_FL_SETTINGS_PLUGIN_H_
+#define FLUTTER_SHELL_PLATFORM_LINUX_FL_SETTINGS_PLUGIN_H_
+
+#include "flutter/shell/platform/linux/public/flutter_linux/fl_binary_messenger.h"
+
+G_BEGIN_DECLS
+
+G_DECLARE_FINAL_TYPE(FlSettingsPlugin,
+                     fl_settings_plugin,
+                     FL,
+                     SETTINGS_PLUGIN,
+                     GObject);
+
+/**
+ * FlSettingsPlugin:
+ *
+ * #FlSettingsPlugin is a plugin that implements FIXME.
```

Comment:
I could really use a plugin that can do that 😀

Reply:
It's on my TODO list.